binary-compatibility-validator/reference-public-api/kotlinx-coroutines-core.txt

```diff
@@ -643,7 +647,6 @@ public abstract interface class kotlinx/coroutines/channels/ReceiveChannel {
 	public abstract fun getOnReceiveOrNull ()Lkotlinx/coroutines/selects/SelectClause1;
 	public abstract fun isClosedForReceive ()Z
 	public abstract fun isEmpty ()Z
-	public abstract fun iterator ()Lkotlinx/coroutines/channels/ChannelIterator;
```

Comment:
Can we have this method in 1.0.0-RC as hidden + LowPriorityInOverloadResolution?
Breaking binary compatibility (especially in iteration over the channel) in RC doesn't seem right

Reply:
Sure. Look at the updated `kotlinx-coroutines-core.txt` please, I am not sure about these changes.